mlrun/datastore/abstract_base.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2025 Iguazio
```

Comment:
Not a good name for the file. There is nothing abstract inside. Maybe remote_client.py?